lumen/ai/agents.py

```diff
@@ -224,12 +223,9 @@ class ChatAgent(Agent):
 
     conditions = param.List(
         default=[
-            "Best for high-level information about data or general conversation",
-            "Can be used to describe available tables",
-            "Use for technical questions about programming, functions, methods, or libraries",
-            "Use for 'how to' questions about specific functions or code usage",
-            "Use for questions about software tools, APIs, or programming concepts",
-            "Not useful for answering data specific questions that require querying tables",
+            "Use for high-level data information or general conversation",
+            "Use for technical questions about programming, functions, methods, libraries, APIs, software tools, or 'how to' code usage",
+            "NOT for data-specific questions that require querying tables",
```

Comment:
Happy to hear your thoughts but maybe we should use "data" instead of "table" everywhere.

```suggestion
            "NOT for data-specific questions that require querying tables",
```